webpack.config.js

```diff
@@ -0,0 +1,26 @@
+var webpack = require('webpack');
+
+module.exports = {
+	debug: true,
+	devtool: '#inline-source-map',
+	resolve: {
+    	extensions: ['', '.webpack.js', '.web.js', '.js']
```

Comment:
Ideally we need to work on the project with code linting and code style.
How about we push for integrating standardjs in a follow-up PR?